statements/src/test/java/org/lukawska/trainsmart/statements/presentation/controller/UserAgreementControllerTest.java

```diff
@@ -0,0 +1,102 @@
+import org.lukawska.trainsmart.statements.application.dto.UserAgreementResponse;
+import org.lukawska.trainsmart.statements.application.services.UserAgreementService;
+import org.lukawska.trainsmart.statements.domain.valueObjects.AgreementStatus;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.ResponseEntity;
+
+import java.util.List;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.lukawska.trainsmart.statements.testutil.StatementTestData.randomStatementCode;
+import static org.lukawska.trainsmart.statements.testutil.UserAgreementTestData.acceptedUserAgreementRequest;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class UserAgreementControllerTest {
+
+    @Mock
+    private UserAgreementService service;
+
+    @InjectMocks
+    private UserAgreementController controller;
+
+    @Test
+    void shouldReturnResponseWhenSignNewAgreementSuccess() {
+        //given
+        UserAgreementRequest request = acceptedUserAgreementRequest();
+        UserAgreementResponse expectedResponse = new UserAgreementResponse(null,
+                                                                           request.userId(),
+                                                                           request.statementCode(),
+                                                                           2,
+                                                                           AgreementStatus.ACCEPTED);
+        when(service.signNewAgreement(request)).thenReturn(expectedResponse);
+
+        //when
+        ResponseEntity<UserAgreementResponse> responseEntity = controller.signNewAgreement(request);
+
+        //then
+        assertThat(responseEntity.getBody()).isNotNull();
+        assertThat(responseEntity.getBody()).isEqualTo(expectedResponse);
+    }
+
+    @Test
+    void shouldReturnResponseWhenReSignAgreementSuccess() {
+        // given
+        UserAgreementRequest request = acceptedUserAgreementRequest();
+        UserAgreementResponse expectedResponse = new UserAgreementResponse(null,
+                                                                           request.userId(),
+                                                                           request.statementCode(),
+                                                                           2,
+                                                                           AgreementStatus.ACCEPTED);
+        when(service.reSignAgreement(request)).thenReturn(expectedResponse);
+
+        // when
+        ResponseEntity<UserAgreementResponse> responseEntity = controller.reSignAgreement(request);
+
+        // then
+        assertThat(responseEntity.getBody()).isNotNull();
+        assertThat(responseEntity.getBody()).isEqualTo(expectedResponse);
+    }
+
```

Comment:
puste linie. Pilnuj tych rzeczy bo zwracam na to uwage za kazdym razem. 
Kiedys moj team leader latwo mnie nauczyl ogarniania pustych linii. Po prostu wypieprzyl moj PR do koszta nad ktorym pracowalem 2 tygodnie i jak reka odjal. :D